css/main.css

```diff
@@ -145,68 +149,77 @@ kbd {
 	display: block;
 }
 
+.flexTopBar {
+	display: flex;
+	flex-direction: column;
+	width: 100%;
+	top: 0;
+	left: 0;
+	z-index: 1;/*Lets topbar render ontop of iframe*/
+	user-select: none;
+}
+
+.spectateBanner::before{
+	content: "👁️ ";
+	font-style: normal;
+}
 .spectateBanner{
 	background-color: #90FF90;
 	cursor: pointer;
-	text-align:center;
+	flex: 0 0 35%;
+	width: 99.5%; /* I think this is due to a padding issue or something? */
+	text-align: center;
+	font-style: italic;
+	background-color: lightgreen;
+	border-style: outset;
+	border-color: lightgreen;
+	font-size: 1em;
+	box-shadow: 0 4px 8px 0 rgba(0,0,0,0.1);
 }
 
 .contentContainer{
 	display: flex;
-	flex-direction: row-reverse;
-	margin-top: var(--topbar-height); /*save room for topbar*/
+	height: 100%;
+}
+
+.sidebarNirnroute{
+	margin-left: 0.5em;
 }
 
 .sidebarContent{
-	width: 500px;
 	display: flex;
 	flex-direction: column;
 	height:100%;
+	overflow: hidden; /* Sidebar links should not scroll with rest of page. */
+}
+#sidebarLinks{
+	padding-left: 0.5em;
+	padding-top: 0.25em;
+	padding-bottom: 0.25em;
+	background: var(--highlight-color);
+	border-bottom: groove;
+	border-width: 2px;
+	border-color: dimgray;
 }
 #sidebarLinks a{
 	text-decoration: none;
 	color: black;
 }
 
-/*this can't be applied to the same as resizableContainer because box-sizing causes the resize widget to be below teh screen.*/
-.screenHeight{
-	position: sticky; /*within the .sidebar, this is the "content" container that slides up and down to stay on the page.*/
-	height: 100vh;
-	top: var(--topbar-height); /*start at very top of page (minus topbar)*/
-	/*the following make the height 100vh minus overlap*/
-	box-sizing:border-box;
-	padding-bottom:3.5em;
-}
-
-/*you can't resize iframes, so we use this to give it a nice resizable handle*/
-.resizableContainer{
-	resize: both;
-	overflow: hidden;
-}
-
-.resizableWidthContainer{
-	resize: horizontal;
-	overflow: hidden;
-}
-
 .iframeContainer{
-	border: 1px solid black;
 	display: flex;
 	flex-direction: column;
-	flex-grow:1;
+	width: 100%;
+	height: 100%;
+	top: 0;
+	left: 0;
```

Comment:
if there's no position style set, we can just leave out top and left.

Reply:
Good point, I'm also noticing that the inspector tool is pointing this out as well, so now I can look for other similar issues like this to remove. All these smaller changes are really good for cleaning the code up so it's more clear what is actually necessary, thanks for pointing all these out.